hummingbot/templates/conf_avellaneda_market_making_strategy_TEMPLATE.yml

```diff
@@ -11,6 +11,15 @@ exchange: null
 # Token trading pair for the exchange, e.g. BTC-USDT
 market: null
 
+# Type of execution timeframe
+execution_timeframe: null
+
+# Start time for running between specified times of a day or between specified dates
+start_time: null
+
+# End time for running between specified times of a day or between specified dates
+end_time: null
+
```

Comment:
Remember to also change the template_version number when changing a template YML file.